control_rate : 50
innfos_can_device : "vcan0"
zltech_can_device : "vcan0"
